src/containers/Society.jsx

```diff
@@ -98,22 +100,27 @@ class Society extends Component {
                 filterActivities={this.filterActivities}
               />
               <div className='activities'>
-                <MasonryLayout
-                  items={
-                    filteredActivities.map(activity => (
-                      <ActivityCard
-                        id={activity.id}
-                        category={activity.category}
-                        date={dateFormatter(activity.date)}
-                        description={activity.activity}
-                        points={activity.points}
-                        status={activity.status}
-                        showUserDetails={showUserDetails}
-                        owner={activity.owner}
-                      />
-                    ))
-                  }
-                />
+                {
+                  societyInfo.requesting ?
+                    <Loader />
```

Comment:
I prefer we go with evaluating this before the return. It is cleaner that way.

or 

```
let {requesting} = societyInfo;
if(requesting){
// place loader here and return to get out of the function.
}
```

Reply:
@kimani-ndegwa I changed the logic for displaying the loader. Although it was done inside the return statement.
Why?
Because I would want to display different elements depending on the value found in `requesting`.